jsonnet/jsonnet.bzl

```diff
@@ -788,11 +788,10 @@ Example:
 
 def jsonnet_repositories():
     """Adds the external dependencies needed for the Jsonnet rules."""
-    http_archive(
-        name = "jsonnet",
-        sha256 = "f6f0c4ea333f3423f1a7237a8a107c589354c38be8a2a438198f9f7c69b77596",
-        strip_prefix = "jsonnet-0.13.0",
-        urls = [
-            "https://github.com/google/jsonnet/archive/v0.13.0.tar.gz",
-        ],
+    git_repository(
+        name = "jsonnet_go",
+        # TODO: Update this once google/go-jsonnet#292 lands.
```

Comment:
NB: I'm publishing this here in advance. Some subsequent coordination is necessary.